.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 script: "./gradlew check integrationTest"
 
 jdk:
-  - openjdk6
+  - openjdk7
```

Comment:
Yes, findbugs requires 1.7 to run. I guess the risk is that we accidentally include some Java 7 API, but using the correct Java level in the IDE project settings and the Android testing should help protect us from that.